routes/keyword.js

```diff
@@ -36,10 +39,17 @@ router.get('/:word', (req, res) => {
   const word = req.params.word
   Keyword.find({ word })
     .lean()
-    .then(words => res.send(words[0].votes))
+    .then((wordList) => {
+      if (wordList.length === 0) {
+        res.status(500).send({
+          err: 'word not found',
+          givens: req.params,
+        })
+      } else res.send(wordList)
+    })
     .catch((err) => {
       res.status(500).send({
-        err: err.toString(),
+        err,
```

Comment:
If you throw an Error, and catch it, it becomes 'err: {}', but if you throw a string, or convert the err to a string, it responds properly.